src/main/java/mate/academy/model/User.java

```diff
@@ -49,4 +50,13 @@ public void setPassword(String password) {
     public void setSalt(byte[] salt) {
         this.salt = salt;
     }
+
+    @Override
+    public String toString() {
+        return "User{"
+                + "id=" + id
+                + ", email='" + email + '\''
+                + ", salt=" + Arrays.toString(salt)
```

Comment:
```suggestion
                + ", salt=" + Arrays.toString(salt)
```

Соль - это чувствительные данные как и пароль, их лучше не включать в `toString()` метод

Reply:
Пофиксил.